content/turing-machines/machines-computations/church-turing-thesis.tex

```diff
@@ -10,15 +10,15 @@
 \olsection{The Church-Turing Thesis}
 
 Turing machines are supposed to be a precise replacement for the
-concept of an effective procedure. Turing took it that anyone who
-grasped the concept of an effective procedure and the concept of a
-Turing machine would have the intuition that anything that could be
-done via an effective procedure could be done by Turing machine. This
-claim is given support by the fact that all the other proposed precise
-replacements for the concept of an effective procedure turn out to be
-extensionally equivalent to the concept of a Turing machine---that is,
-they can compute exactly the same set of functions. This claim is
-called the \emph{Church-Turing thesis}.
+concept of an effective procedure. Turing took it to mean that anyone
```

Comment:
What's meant here is that Turing thought that ... ("took it to mean that" requires a referent for "it" and it's not clear what that would be in this case)

Reply:
So "Turing thought that anyone who grasped both..." should be good then?